etc/config.yaml

```diff
@@ -139,6 +139,18 @@ dependencies:
   shapelet:
     run:   [ numpy ]
     build: [ numpy ]
+  mariadb:
+    run:   [ openssl ]
+    build: [ openssl ]
+  mariadb:
```

Comment:
err, uhh... I'm guessing one of those was supposed to be `mariadbclient`, which is about to be added to the build in place of `mysqlclient`.